src/fixed-footer-layout.tsx

```diff
@@ -68,6 +68,11 @@ const FixedFooterLayout: React.FC<Props> = ({
     }, [onChangeFooterHeight, domFooterHeight]);
 
     React.useEffect(() => {
+        // There is no elevation in desktop devices, so we avoid unnecesary calculations
+        if (!isTabletOrSmaller) {
```

Comment:
This was not strictly necessary, but we can avoid all these calculations if we are in a desktop device

Reply:
Could you add this check to `shouldDisplayElevation` function and not do the calculations for the other cases too? Just wondering.

Reply:
Discussed offline. Refactored this useEffect to make it cleaner